cmd/expiration-mailer/main.go

```diff
@@ -480,6 +453,107 @@ func (m *mailer) findExpiringCertificates(ctx context.Context) error {
 	return nil
 }
 
+func (m *mailer) getCerts2(ctx context.Context, left, right time.Time, expiresIn time.Duration) ([]certDERWithRegID, error) {
+	// First we do a query on the certificateStatus table to find certificates
+	// nearing expiry meeting our criteria for email notification. We later
+	// sequentially fetch the certificate details. This avoids an expensive
+	// JOIN.
```

Comment:
This comment hasn't been updated to reflect the JOINyness of this function. Arguably both this and the similar comment on `getCerts` should be doccomments rather than inline comments.